example_sns_init.yaml

```diff
@@ -112,9 +113,31 @@ Swap:
     restricted_countries:
         - AQ  # Antarctica
 
+    # Declare that each participant will get three SNS neurons in the following
+    # states:
+    # 1. Unlocked
+    # 2. Locked for 1 month
+    # 3. Locked for 2 months
     VestingSchedule:
-        events: 104
-        interval: 7 d
+        events: 3
+        interval: 1 month
+
+    # WARNING: Swap start_time works differently on mainnet and in testing.
+    #
+    # On mainnet:
+    # - Setting start_time to some value (e.g., 12:00 UTC) will allow the swap
+    #   participants to be prepare for the swap in advance, e.g.,
+    #   by obtaining ICPs that they would like to participate with.
+    # - If start_time is not specified, the actual start time of the swap will
+    #   be chosen at random. TODO: Explain how this works.
```

Comment:
Proposed explanation:

```
Note that an algorithm will be applied to allow
    #   at least 24 hours before the swap starts. For example, setting this
    #   value to 23.59 UTC implies that the swap will actually start in 47:59
    #   two days minus one minutes. 
```

Reply:
It seems that your proposed explanation refers to the case when `start_time` is set, but the TODO left refers to the case when no `start_time` is provided in the yaml file. Moreover, I don't think that your text describes how the actual swap start time is determined: the `start_time` is an absolute time within a day (that's why there's also a time zone) rather than an offset to be applied to the 24-hour window (please also see my example in a thread further above).

Reply:
Edited. Please proof read this comment again.

Reply:
Thank you! Added a few more comments.